cmd/manager/main.go

```diff
@@ -118,6 +120,15 @@ func main() {
 		setupLog.Error(err, "unable to create controller", "controller", "FluentBit")
 		os.Exit(1)
 	}
+
+	if err = (&controllers.ClusterFluentBitConfigReconciler{
```

Comment:
Move up before the FluentBitReconciler is set up.

Reply:
thanks，Most of this is auto-generated, and I've fixed it